source/kraken/maintenance_worker.cpp

```diff
@@ -472,6 +503,8 @@ std::vector<AmqpClient::Envelope::ptr_t> MaintenanceWorker::consume_in_batch(con
             ++consumed_nb;
         }
     }
+    LOG4CPLUS_DEBUG(logger, "It took " << (pt::microsec_clock::universal_time() - begin).total_milliseconds()
+                                       << "ms to retrieve " << envelopes.size() << " message from the broker");
```

Comment:
I also added `(s)` to `message` since it could be more than one

```suggestion
    LOG4CPLUS_DEBUG(logger, "retrieving " << envelopes.size() << " message(s) from the broker took " << (pt::microsec_clock::universal_time() - begin).total_milliseconds()
                                       << "ms");
```